backend/core/settings.py

```diff
@@ -72,7 +78,6 @@
     "content",
     "events",
     "rest_framework_simplejwt",
-    "django_filters",
```

Comment:
question: Where did `django_filters` go in this, @Jolanto? Was this removed on accident?